docs/usage/network_graph.md

```diff
@@ -0,0 +1,21 @@
+# Network graph
+
+Network graph allows you to visualize your Zwave mesh network and check connectivity issues between nodes and the controller.
+
+## Nodes Healthcheck
+
+In order to use this feature you need to open **Network Graph** tab, apply some filters in case you have big graph to only see required nodes and then click on `Reload Graph`. At this point you can see your nodes and how them are connected to the controller.
```

Comment:
```suggestion
In order to use this feature you need to open **Network Graph** tab. If you have a large network, you might want to apply some filters to only see the relevant nodes. Then click on `Reload Graph`. At this point you can see your nodes and how they are connected to the controller.
```
I'm not happy with
> how they are connected to the controller

though, since we know this information is often plain wrong.

Reply:
And so what?

Reply:
Probably just remove the last sentence

Reply:
Check now